### PR TITLE
check image versions

### LINKED ISSUE #838: Suggest users to upgrade Kusk's cluster deployments
Currently we just suggest to them to update the CLI. 

Can we add a timely suggestion to nudge users to run `kusk cluster upgrade` so they can actually reap the new features and not just installing the CLI?

### PR DESCRIPTION
**Pull request by jasmingacic**
This PR allows to review which images have been outdated in the cluster by running `kusk version` 

image

Fixes #838 
Signed-off-by: jasmingacic <jasmin.gacic@gmail.com>

### CODE REVIEW

**Comment by donnalux** [comment]
@jasmingacic did a quick mock up for this so it would follow same pattern as the upgrade CLI messaging.
https://www.figma.com/file/EkNk9tMEoAgYrwGP7N1dPm/Kusk-CLI?node-id=359%3A375

**Comment by jasmingacic** [comment]
image

**Comment by jasmingacic** [comment]
Fixed wording a little
image

File: cmd/kusk/cmd/version.go [hunk 1]
```diff
@@ -89,20 +89,85 @@ func NewVersionCommand(writer io.Writer, version string) *cobra.Command {
...
+	// fmt.Println("repo", repo, current)
+	// if len(current) > 0 {
+	latestImg, err := ghClient.GetLatest(repo)
+	if err != nil {
+		return status, err
+	}
+
+	status.Latest = latestImg
+	status.Outdated = !utils.IsUptodate(latestImg, current)
+	// }
```
Comment (kylehodgetts) [inline cmd/kusk/cmd/version.go:161]:
```suggestion
```

**Review by aabedraba — APPROVED** [review]

**Pull request merged**